code/workshops.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Coding Club for Girls</title>
+    <link rel="stylesheet" type"text/css" href="style.css">
+  </head>
+
+ 
+
+  <body>
+    <header>
+        <div class="header">
+            <img src="Logo1.png" alt="Logo Coding Club">
+            <h1>Coding Club for Girls</h1>
+        </div>
+    </header>
+
+    <menu>
+      <a href="./index.html">Home</a>
+      <a href="./workshops.html">Workshops</a>
+      <a href="https://www.bfh.ch/codingclub">Über uns</a>
+      <a href="https://www.epfl.ch/education/education-and-science-outreach/fr/promotion-education-sciences/promotion-des-sciences/coding-club-for-girls/">Partner</a>
+    </menu>
+
+    <h2>Unsere Workshops</h2>
+        <div class="container">
+      
+            <div class="box01">
+                <div class="text_center">
+                    <p>
+                        <h3>Sei dabei im Coding Club - jetzt Anmelden!</h3>
```

Comment:
Same here with keeping the `<h3>` outside of the `<p>` element.

Reply:
Ragna, thank you so much for your kind words and your helpful tips!